vignettes/performance_analysis.Rmd

```diff
@@ -0,0 +1,323 @@
+---
+title: "Conducting Performance Analyses"
+output: rmarkdown::html_document
+vignette: >
+  %\VignetteIndexEntry{Conducting Performance Analyses}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+This vignette introduces the concept of a performance analysis and demonstrates how `eiCompare` can be used to conduct one given voter files and districting maps.
+
+```{r imports, echo=FALSE, message=FALSE, warning=FALSE}
+suppressMessages({
+  library(dplyr)
+  library(eiCompare)
+  library(ggplot2)
+  library(sf)
+  library(wru)
+})
+```
+
+## What is a performance analysis? 
+
+A successful voting rights case may result in a local jurisdiction's districting map to be thrown out, prompting the need for a new map. Thus, it's necessary assess whether the new map provides sufficient representation for minority groups. It’s not desirable to simply wait for elections to happen and see what the results might be. Instead, we can look at past elections, and observe how candidates would perform if the proposed map was used at the time of the election. This is the basis of a performance analysis.
+
+Ultimately, we need to assess the demographic breakdown across racial groups for each district in the new map. What data source do we use to calculate the percentage of each racial group? We could use total population or citizen voting age population (CVAP), as provided by the Census Bureau. However, these measures may not be accurate, because they assume that voter turnout is equal across racial groups. This is not always true, especially in cases of racial gerrymandering, where turnout for minority groups may be depressed. Instead, we argue that going to the level of the _voter file_ is necessary, as the voter file actually informs _who turned out to vote_.
+
+## The steps in a performance analysis
+
+To conduct a performance analysis, we need to join the voters in a voter file to the new districts and determine the turnout by race, per district. This includes the following steps:
+
+1. **Geocoding:** Geocode latitude and longitude of voters in the file using their address. We do not focus on this step in this vignette.
```

Comment:
In the website version (if we're permitting ourselves small differences), could link to Juandalyn's vignette here.